org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/ModelReflection.java

```diff
@@ -6,20 +6,20 @@
 import org.eclipse.scanning.api.INameable;
 
 /**
- * 
+ *
  * @author Matthew Gerring
  *
  */
 public class ModelReflection {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)